docs/io.cozy.accounts.md

```diff
@@ -25,19 +25,11 @@ Accounts can be managed in [Cozy-Home](http://github.com/cozy/cozy-home/) (via [
 
 ### `auth`
 
-The `auth` attribute may also contain other data, like `accountName`, `folderPath` or `frequency`. As `auth` should only be used for authentication mechanisms, those two values should disappear soon.
-
-- `folderPath` should purely and simply disappear, the folder information is stored in `io.cozy.trigger`.
-- `accountName` is in reality the `label`, change should be made soon to fix this mistake.
-- `frequency` should move at the root of the account.
-- `namePath`: The name of the last folder. Usually saved along with `folderPath`.
+- `login` main credentials fields, displayed as default accountName. Some konnectors do not use a `login` parameter, but `identifier` or `email`. In this case the attribut role: 'identifier' should be add to parameter object in manifest.
```

Comment:
```suggestion
- `login` main credentials fields, displayed as default accountName. Some konnectors do not use a `login` parameter, but `identifier` or `email`. In this case the attribut role: 'identifier' should be added to parameter object in manifest.
```

Reply:
Also, login is a string, right? 

Can you do like above? Something like: 
```
`login`: {string?} Main credential field.
`identifier`: {string?} If the konnector does not use `login` field. In that case, this attribute should declared as `role: identifier` in the manifest.
`email`: {string?} If the konnector does not use `login` or `identifier` fields. In that case, this attribute should declared as `role: identifier` in the manifest. 
```

Reply:
/attribut/attribute/